core/src/main/java/com/alibaba/nacos/core/controller/NacosClusterController.java

```diff
@@ -103,6 +110,7 @@ public RestResult<String> getHealth() {
      */
     @Deprecated
     @PostMapping(value = {"/report"})
+    @Secured(resource = Commons.NACOS_CORE_CONTEXT + "/cluster", action = ActionTypes.WRITE, signType = SignType.CONSOLE)
```

Comment:
This api is called by servers, you should make sure the api can be called success after you add this.

Reply:
method reportByHttp will call AuthHeaderUtil.addIdentityToHeader(header);
![image](https://github.com/alibaba/nacos/assets/73023513/5f13a426-ab3d-4752-9018-8336bff022a2)